validator/build.gradle

```diff
@@ -55,6 +59,10 @@ dependencies {
     implementation 'com.amazonaws:aws-java-sdk-cloudwatch'
     implementation 'com.amazonaws:aws-java-sdk-xray'
 
+    // https://mvnrepository.com/artifact/com.github.awslabs/aws-request-signing-apache-interceptor
+    implementation 'com.github.awslabs:aws-request-signing-apache-interceptor:b3772780da'
```

Comment:
what's "b3772780da"?

Reply:
It's the version of the library in maven: https://mvnrepository.com/artifact/com.github.awslabs/aws-request-signing-apache-interceptor?repo=mulesoft-public.